bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/servlets/AllowedHeadingElementsDataSourceServlet.java

```diff
@@ -52,6 +52,7 @@
     public final static String RESOURCE_TYPE_TITLE_V1 = "core/wcm/components/title/v1/datasource/allowedtypes";
     public final static String PN_ALLOWED_HEADING_ELEMENTS = "allowedHeadingElements";
     public final static String PN_ALLOWED_TYPES = "allowedTypes";
+    public final static String PN_DEFAULT_TYPE = "type";
```

Comment:
@pauli, we should add `PN_DEFAULT_HEADING_ELEMENT = "headingElement";` and test this also works in the accordion component.